### ISSUE TITLE
Accuracy about ResNet101V1c (backbone) on ImageNet-1k used in DeepLabv3+

### ISSUE THREAD

**Issue by xiaoachen98** [open-mmlab/mmsegmentation#1176]
Could you provide  the accuracy of the backbone (Resnet101v1c) used in DeepLabV3 +  pretrained on ImageNet1k?

**Comment by MengzhangLI** [comment]
Hi, sorry to tell you that it is not provided yet, because currently logits value is not directly passed to metric calculation function.

You can refer to this [issue](https://github.com/open-mmlab/mmsegmentation/issues/1157) if you really want to calculate accuracy.

Best,

**Comment by xiaoachen98** [comment]
I mean the accuracy on the ImageNet1k val dataset calculated by the Resnet101v1c weights which you used as the pretrained weights to initialize.

**Comment by MengzhangLI** [comment]
> I mean the accuracy on the ImageNet1k val dataset calculated by the Resnet101v1c weights which you used as the pretrained weights to initialize.

I see, sorry for misleading explanation. Its actual link is here:

https://github.com/open-mmlab/mmcv/blob/d30e37d4d57425843bbff484e7e7a1e41230381f/mmcv/model_zoo/open_mmlab.json#L39

As far as I know, it is from MSRA. But I dont know its detailed results on ImageNet1K.
